### DISCUSSION TITLE
Internet on wlan1

### DISCUSSION THREAD

**Discussion by Bismaeck** [RaspAP/raspap-webgui#1258] [Q&A]
Hallo,
My Problen is Internet on wlan1...

My Configuration:
- Raspberry Pi 4b
- Raspbian bullseye lite 64bit
- RaspAP 2.8.6
  = RaspAP is DHCP Server on wlan0 (1.8.5.1) and eth0 (1.12.5.1)
     => that works for my needs but additional I want to link my raspi
	   with an existing wlan Netork (for Internetaccess).
	   Therefor I use an Edimax EW-7811UN V1 as wlan1.
	   RaspAP ist connected to the other network on wlan1. So far so good...
	
	BUT THERE'S NO INTERNETACCESS!!!

Has Anyone any idea?

**Comment by billz** [comment]
This is a very common question. Step-by-step instructions are provided in [this FAQ](https://docs.raspap.com/faq/#interfaces).
Also discussed in #1022 and #1171.
